demo/server.jsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * Created by avim on 6/27/16.
+ */
+
+import {render} from 'ReactOverTheWireDOM';
+import nativeExtensions from './nativeExtensions';
+
+function renderServer(targetId) {
+    var ws = new WebSocket('ws://localhost:1234', window.chosenDemo);
+    ws.addEventListener("open", () => {
+
+        render({
+            addEventListener: (handler) => {
+                ws.addEventListener("message", (e) => {
+                    handler(JSON.parse(e.data));
```

Comment:
We may not need to stringify for websockets. I had used `JSON.parse` and `JSON.stringify` because that was the fastest way for transferring objects in web workers. For web sockets, plain Javascript objects may be faster.